index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="Main.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1 class="Header">Hello Microverse!</h1>
```

Comment:
[OPTIONAL] The industry standard is to lowercase class names, I recommend you fix this.

Reply:
Done